ts/api/api.ts

```diff
@@ -123,6 +124,11 @@ export interface OpenYoloApi {
    * Cancels the last pending OpenYOLO request.
    */
   cancelLastOperation(): Promise<void>;
+
+  /**
+   * Sets the desired log level for the OpenYOLO library.
+   */
+  setLogLevel(level: LogLevel): Promise<void>;
```

Comment:
Maybe I am missing something. The SPI has access to those symbols anyway right ? We export everything under `protocol/data` and the provider frame can choose to implement the API as this is one of the requests that can be send by `api.ts`.